### PR TITLE
Preparing release v1.37.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

## Short description of the changes
 - https://github.com/jaegertracing/jaeger-operator/issues/2017

@iblancasa  as soon as the openshift test pass we can merge this.

//cc @pavolloffay

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+1.37.0 (2022-08-11)
+-------------------
+
+* Upgrade operator-sdk to 1.22.2 ([#2021](https://github.com/jaegertracing/jaeger-operator/pull/2021), [@iblancasa](https://github.com/iblancasa))
+* Bump github.com/prometheus/client_golang from 1.12.2 to 1.13.0 ([#2019](https://github.com/jaegertracing/jaeger-operator/pull/2019), [@dependabot[bot]](https://github.com/apps/dependabot))
```
Comment (frzifus) [inline CHANGELOG.md:8]: Do we need to mention this?

**Comment by iblancasa** [comment]
@rubenvp8510 could you rerun the failed workflows? I'm not sure what happened there.

File: CHANGELOG.md (hunk 1 above)
Comment (rubenvp8510) [reply CHANGELOG.md:8]: not really

**Review by iblancasa — CHANGES_REQUESTED** [review]
I ran the E2E tests on OCP 4.10. All them passed.
Please, do not merge until all the tests are passed here using kind.

**Review by iblancasa — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 Changes by Version
 ==================
 
+1.37.0 (2022-08-11)
+-------------------
+
+* Upgrade operator-sdk to 1.22.2 ([#2021](https://github.com/jaegertracing/jaeger-operator/pull/2021), [@iblancasa](https://github.com/iblancasa))
+<<<<<<< HEAD
```
Comment (iblancasa) [inline CHANGELOG.md:8]: It seems the merge was not done properly. Could you take a look? @rubenvp8510

**Review by iblancasa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (rubenvp8510) [reply CHANGELOG.md]: Oops, forgot to save the file. Fixed

Thanks!

**Review by iblancasa — APPROVED** [review]

**Review by frzifus — APPROVED** [review]

**Pull request merged**